docs/guides/medical.md

```diff
@@ -0,0 +1,181 @@
+# ArmaForces - Medical
+
+## Menu Medyczne
+
+### Ustawienie sterowania
+
+Do obsługi Medicala należy wpierw mieć zbindowane menu medyczne. Istnieje możliwość udzielania pierwszej pomocy za pomocą ACE, ale jest to nieporęczne. Na poniższych obrazkach przedstawiono w jaki sposób możemy zbindować szybki dostęp do menu medycznego. 
+
+![Ustawienie bindu menu medycznego](../_data/guides/medical/key_medical_menu.png)
+
+### Obsługa Medicala
+
+Po kliknięciu przypisanego klawisza (domyślnie `H`) wyświetli nam się menu medyczne nasze osoby, na którą mamy skierowany kursor i jest wystarczająco blisko. Jeśli w nikogo nie celujemy wybrana zostaje nasza postać.
+
+![Menu medyczne](../_data/guides/medical/medical_menu.png)
+
```

Comment:
I would be nice to have some quick introduction because you have some bullet points but it took me a while to undestand what does that even mean. Maybe whole paragraph will need some rephrasing to make it less out of context.

I'm thinking now while reviewing and maybe the bullet points should be separated from numbered list. I'd make one screenshot which has medical menu areas numbered (to allow people to easier connect the bullet point to the screenshot) and second one with small icons numbered (as they are now but only the icons, without whole screenshot). If something is unclear I can elaborate more.